.github/workflows/ci.yml

```diff
@@ -944,12 +944,12 @@ jobs:
     strategy:
       matrix:
         job:
-          - os: ubuntu-20.04
+          - os: ubuntu-22.04
```

Comment:
current/latest ubuntu LTS seems to be `24.04`; shall we not use that?

Reply:
https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories

Reply:
approving since don't this to be a blocker; can re-approve if you update aswell

Reply:
We will move it off this runner anyways, let's merge this one and we will be using `ubuntu-latest` once the migration is done: https://github.com/FuelLabs/sway/issues/7102